scss/civicrm/common/_modals.scss

```diff
@@ -48,18 +47,16 @@
       background: none;
       font-size: $font-size-h2;
       font-weight: $crm-font-weight-h2;
+      line-height: $font-size-h1;
       padding-left: 0;
     }
 
     > .ui-button {
       &[type="button"] {
+        margin-top: 0;
+        padding: 0;
```

Comment:
I am thinking, shouldn't it be 
`top: 0; right: 0; padding: $modal-block-padding;` where `$modal-block-padding=13px 20px;`

That will also make a button easier to click.

Reply:
that sounds like the most elegant solution, but it breaks the remaining icons:

![screenshot 2017-10-02 07 13 53](https://user-images.githubusercontent.com/1642119/31074994-c2b33478-a71f-11e7-96aa-b08347ef2ade.png)

Reply:
Hm. It doesn't for me. Padding doesn't actually work since it mixes with height/width, but margin does nicely:

![image](https://user-images.githubusercontent.com/3973243/31075245-7be1344e-a76c-11e7-965e-b7523b252dc4.png)

Reply:
@reneolivo anyway, if it doesn't really work, that can be skipped for now. I am just trying to suggest to get rid of as many constants as possible.